charts/datarepo-api/templates/api-deployment.yaml

```diff
@@ -13,8 +13,11 @@ spec:
   template:
     metadata:
       labels:
+{{- include "datarepo-api.labels" . | nindent 8 }}
```

Comment:
Would you kindly add a comment on why this is necessary?

Reply:
so you can search by labels in kubectl 

```
kubectl get pods --namespace temp -l app.kubernetes.io/name=oidc-proxy    
```